### PR TITLE
Better fix for #1641.

### PR DESCRIPTION
**Pull request by vdbergh**
Again not tested.

### CODE REVIEW

File: server/fishtest/rundb.py [hunk 1]
```diff
@@ -861,6 +861,10 @@ def priority(run):  # lower is better
         for task in run["tasks"]:
             opening_offset += task["num_games"]
 
+        if "sprt" in run:
```
Comment (ppigazzini) [inline server/fishtest/rundb.py:864]:
```python
if "sprt" in run["args"]:
    sprt_batch_size_games = 2 * run["args"]["sprt"]["batch_size"]
    remaining = sprt_batch_size_games * math.ceil(remaining / sprt_batch_size_games)

```

Comment (vdbergh) [reply server/fishtest/rundb.py:864]: Thanks! Sorry about that.... I will fix it.

Comment (vdbergh) [reply server/fishtest/rundb.py]: Done...

**Review by ppigazzini — APPROVED** [review]
Looks good on DEV

**Comment by ppigazzini** [comment]
PROD updated, thank you @vdbergh :)

**Comment by dav1312** [comment]
@vdbergh you can use `Fixes #1641` in the PR description so the issue is closed automatically
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue

**Pull request merged**